server/index-node/src/resolver.rs

```diff
@@ -149,24 +165,135 @@ where
             .map(|info| info.into_value())
             .unwrap_or(q::Value::Null))
     }
+
+    fn resolve_subgraph_features(
+        &self,
+        arguments: &HashMap<&str, q::Value>,
+    ) -> Result<q::Value, QueryExecutionError> {
+        // We can safely unwrap because the argument is non-nullable and has been validated.
+        let subgraph_id = arguments.get_required::<String>("subgraphId").unwrap();
+
+        // TODO:
+        //
+        // An intersting optimization would involve trying to get the subgraph manifest from the
```

Comment:
minor typo, from `intersting` to `interesting`.

Also, great comment here for the future! 👌